providers/os/resources/windows/registrykey_powershell.go

```diff
@@ -69,131 +61,3 @@ ConvertTo-Json -compress $properties
 func GetRegistryKeyChildItemsScript(path string) string {
```

Comment:
should we move this code to the registry package as well or was there a reason why we left this out?

Reply:
yep, moved. it was a leftover, let's keep all registry powershell related stuff in one place